content/registration.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Registration"
+weight: 3
+type: page
+menu: main
+---
+
+Registration will open on the 20th of September and will close on **Friday the 3rd of October**.
+Registration will be done through the [ICPC system](https://icpc.global/regionals/finder/BAPC-2026).
+Teams can be promoted from the preliminaries.
+Teams will be accepted once the registration is closed, on Saturday the 4th of October.
+Please don't register more than five teams per institution.
+
+## Team Admission rules
+
+The following rules will be adhered to when accepting teams. Please note that based on previous years, we expect that
+at least three teams per institution should be no problem.
+
+* The maximum number of teams (50) will not be exceeded.
+* Every institution will be guaranteed one team.
+* `n`th teams will be accepted based on preliminary results.
+    - The `n`th teams will be awarded round-robin per institution.
+    - Teams will be prioritized based on the highest number of problems solved (or if this is equal, the lowest time penalty) in the preliminary.
+    - In case of a tie, either both teams get a slot or neither team gets a slot.
+    - Institutions without a preliminary will be awarded extra slots last. The coach may in this case provide an order for the teams to be accepted in.
+    - No institution gets a `n+1`th team slot until all registrations of `n`th team of an institution is accepted.
+* No more than five teams per institution will be accepted.
```

Comment:
Do we need to state this? This gives a (very small) probability that we'll have less than 50 student teams.

On the other hand, it is good to explicitly state that each institution should register at least 5 teams (and perhaps even more), to make sure that none of their teams miss out on the fun, just because other institutions happened to register more teams.

Reply:
I did 5 since that was the number of maximum accepted teams in the last few years.  I'll change it to six.

Reply:
> The comment about encouraging the institutions to actually register teams for all of their allowed six slots still holds.
This is already mentioned in the paragraph above that. There is no need to expand this, since this was also never a problem in the past.